bookings/views.py

```diff
@@ -1,3 +1,84 @@
-from django.shortcuts import render
+import json
+from datetime         import datetime
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+from django.db.models import Q
+
+from .models          import Booking
+from products.models  import Product
+from users.utils      import login_decorator
+
+class BookingView(View):
+    @login_decorator
+    def post(self, request, product_id):
+        try:
+            data        = json.loads(request.body)
+            
+            check_in    = data.get("checkin")
+            check_out   = data.get("checkout")
+            head_count  = data.get("count")
+            
+            if not check_in or not check_out or not head_count:
+                raise KeyError
+            
+            first  = datetime.strptime(check_in, "%Y-%m-%d")
+            second = datetime.strptime(check_out, "%Y-%m-%d")
+            within = second - first
+            
+            price       = Product.objects.get(id=product_id).price
```

Comment:
여기서, product_id에 매칭되는 데이터가 없는 경우는 어떤 에러가 발생하나요?

```suggestion
            price       = Product.objects.get(id=product_id).price
```